### PR TITLE
Update receipt/reminder email contents for counselor and admin courses

### PR DESCRIPTION
**Pull request by davidsbailey**
axosoft item: https://codeorg.axosoft.com/ViewItem.aspx?type=features&id=416

new reminder text: https://docs.google.com/document/d/13gRZZ8-5ikwe99l-QItVLTkIxAuX9bE8VEkx7Iakof8/edit#

the receipt email text uses the same text as the reminder email text, except instead of "This is a reminder about your upcoming Code.org Administrator/Counselor workshop." it will say "Thanks for enrolling in Code.org’s Administrator/Counselor workshop!" here are some examples of how this looks now:
## counselor receipt email

screen shot 2016-10-12 at 12 30 10 pm
## administrator reminder email

screen shot 2016-10-12 at 12 56 03 pm

### CODE REVIEW

File: dashboard/app/models/pd/workshop.rb [hunk 1]
```diff
@@ -183,6 +183,10 @@ def self.in_state(state)
     end
   end
 
+  def course_name
+    course == Pd::Workshop::COURSE_ADMIN ? 'Administrator' : course
+  end
+
```
Comment (aoby) [inline dashboard/app/models/pd/workshop.rb:200]: This seems hacky.

If we want these to be called `Administrator` instead of `Admin`, shouldn't that be across the board? We can change the constant value and upgrade existing data in a migration (which should be a separate PR, run before this).

Comment (aoby) [reply dashboard/app/models/pd/workshop.rb:200]: Otherwise it will show up as `Admin` in the UI, and `Administrator` in the emails.

Comment (davidsbailey) [reply dashboard/app/models/pd/workshop.rb:200]: I agree this is hacky. I also think it's valuable separate the keys we use in the database from the exact text we show on the screen, in fact in most places we are forced to do this so we can translate the strings. How about something like this?

```
COURSE_NAMES = {
  Pd::Workshop::COURSE_ADMIN: 'Administrator',
  Pd::Workshop::COURSE_COUNSELOR: 'Counselor',
  ...
}
def course_name
  COURSE_NAMES[course]
end
```

Comment (aoby) [reply dashboard/app/models/pd/workshop.rb:200]: So far this is US-only and not translated, but that's a good point. I like this better. 

Ideally we'd have a similar thing for subject, and this name would be returned in the API (to be displayed in the UI) instead of the DB value.
